benchalerts/workflows.py

```diff
@@ -115,7 +115,10 @@ def update_status(description, state, details_url):
         regressions = benchmarks_with_z_regressions(all_comparisons)
         log.info(f"Found the following regressions: {regressions}")
 
-        if regressions:
+        if not all_comparisons:
+            desc = "Could not find any baseline runs to compare to"
+            state = github.StatusState.SUCCESS
```

Comment:
We don't need to do this now + we might only be able to when we move to checks, but this should probably be an info or warning state and not success I suspect

Reply:
Agreed - like you said though the only options on this endpoint are PENDING, ERROR, FAILURE, or SUCCESS.